source/retryable-reads/tests/changeStreams-client.watch-serverErrors.yml

```diff
@@ -0,0 +1,144 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+    
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "client.watch succeeds after InterruptedAtShutdown"
```

Comment:
Similar comment regarding tests for expecting errors on `retryReads: false` and after two successive server errors. I'll omit repeating this comment in future test files, as I imagine it applies to all read operations (not just `aggregate` and change streams).